server/schedulers/evict_leader.go

```diff
@@ -89,13 +143,19 @@ func newEvictLeaderScheduler(opController *schedule.OperatorController, conf *ev
 	}
 
 	base := newBaseScheduler(opController)
+	handler := newEvictLeaderHandler(conf)
 	return &evictLeaderScheduler{
 		baseScheduler: base,
 		conf:          conf,
 		selector:      selector.NewRandomSelector(filters),
+		handler:       handler,
 	}
 }
 
+func (s *evictLeaderScheduler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
+	s.handler.ServeHTTP(w, r)
+}
+
 func (s *evictLeaderScheduler) GetName() string {
```

Comment:
there is a [race issue](https://internal.pingcap.net/idc-jenkins/blue/organizations/jenkins/pd_test/detail/pd_test/2894/pipeline).